### ISSUE TITLE
Use as low-latente voice audio streamer

### ISSUE THREAD

**Issue by rvt** [Johni0702/mumble-web#79]
Hey,

Would it possible to use this client striped down for audio streaming only? No chat, no mic...
I am looking for a low latency way to stream voice audio (airband, from RTL-SDR) from my server to attached web clients.

**Comment by Johni0702** [comment]
Yes, totally. This fork does something similar (as far as I understand from the commit messages, haven't actually seen it running): https://github.com/c3lingo/mumble-web
